### DISCUSSION TITLE
Add a signiture as picture

### DISCUSSION THREAD

**Discussion by TheuRe** [posquit0/Awesome-CV#605] [General]
Hey,
I would like to add a signiture in the coverletter as any picture/scanned file.
If I add a picture with `\includegrafics{xxxx.xxx}` I get a bad scaling and spacing, anywhere I include the command.
Is there any command already included or how can I can manage the issue?!
Regards
TheuRe

**Comment by johannesbottcher** [comment]
`\letterclosing{Sincerely,\\\includegraphics[scale=.07]{example-image}\\[-2\baselineskip]}`

**Comment by TheuRe** [comment]
That is the solution, thank you!
